benchmark/Jenkinsfile

```diff
@@ -56,34 +57,56 @@ pipeline {
 
      silentResponse: false,
 
-     regexpFilterText: '$comment',
-     regexpFilterExpression: '@JSOBot runbenchmarks'
+     regexpFilterText: '$comment $pullrequest',
+     regexpFilterExpression: '@JSOBot runbenchmarks( .*\\.jl)? ' + prNumber
     )
   }
   stages {
     stage('clone repo') {
       when {
-        expression { REPO_EXISTS == 'false' }
+        expression { REPO_EXISTS == 'false' && env.comment }
       }
       steps {
         sh 'git clone https://${GITHUB_AUTH}@github.com/$org/$repo.git'
       }
     }
     stage('checkout on new branch') {
+      when {
+        expression { env.comment }
+      }
       steps {
         dir(WORKSPACE + "/$repo") {
           sh '''
-          git clean -fd
-          git checkout main
-          git pull origin main
-          git fetch origin
-          git branch -D $BRANCH_NAME || true
-          git checkout -b $BRANCH_NAME origin/$BRANCH_NAME
+            git clean -fd
+            git checkout main
+            git pull origin main
+            git fetch origin
+            LOCAL_BRANCH_NAME="temp_bmark"
+            git branch -D $LOCAL_BRANCH_NAME || true
+            git fetch origin pull/$pullrequest/head:$LOCAL_BRANCH_NAME
+            git checkout $LOCAL_BRANCH_NAME --
           '''
         }
+        sh "git clone https://github.com/ProofOfConceptForJuliSmoothOptimizers/BenchmarkSetup.git || true"
```

Comment:
org name here

Reply:
At the moment, this repository contains all the scripts that run the benchmarks.